doc/doc-site/docs/off-chain/dss/gcp.md

```diff
@@ -0,0 +1,916 @@
+:::
+
+## Initial setup
+
+Follow the steps in this section in order, making sure to keep the relevant shell variables stored in your active shell session.
+
+:::tip
+Use a scratchpad text file to store shell variable assignment statements that you can copy-paste into your shell:
+
+```sh
+ORGANIZATION_ID=123456789012
+BILLING_ACCOUNT_ID=ABCDEF-GHIJKL-MNOPQR
+REGION=a-region
+ZONE=a-zone
+```
+
+:::
+
+### Configure project
+
+1. [Create a GCP organization](https://cloud.google.com/resource-manager/docs/creating-managing-organization), [try GCP for free](https://cloud.google.com/free), or otherwise get access to GCP.
+
+1. [Install the Google Cloud CLI](https://cloud.google.com/sdk/docs/install-sdk).
+
+1. List the [organizations](https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#organizations) that you are a member of:
+
+   ```sh
+   gcloud organizations list
+   ```
+
+1. Store your preferred organization ID in a shell variable:
+
+   ```sh
+   ORGANIZATION_ID=<YOUR_ORGANIZATION_ID>
+   ```
+
+   ```sh
+   echo $ORGANIZATION_ID
+   ```
+
+1. Choose a project ID (like `fast-15`) that complies with the [GCP project ID rules](https://cloud.google.com/sdk/gcloud/reference/projects/create) and store it in a shell variable:
+
+   ```sh
+   PROJECT_ID=<YOUR_PROJECT_ID>
+   ```
+
+   ```sh
+   echo $PROJECT_ID
+   ```
+
+1. Create a new [project](https://cloud.google.com/storage/docs/projects) with the name `econia-dss`:
+
+   ```sh
+   gcloud projects create $PROJECT_ID \
+       --name econia-dss \
+       --organization $ORGANIZATION_ID
+   ```
+
+1. List your billing account ID:
+
+   ```sh
+   gcloud alpha billing accounts list
+   ```
+
+   :::tip
+   As of the time of this writing, some billing commands were still in alpha release.
+
+   If you prefer a stable command release, you might not need to use the `alpha` keyword.
+   :::
+
+1. Store the billing account ID in a shell variable:
+
+   ```sh
+   BILLING_ACCOUNT_ID=<YOUR_BILLING_ACCOUNT_ID>
+   ```
+
+   ```sh
+   echo $BILLING_ACCOUNT_ID
+   ```
+
+1. Link the billing account to the project:
+
+   ```sh
+   gcloud alpha billing projects link $PROJECT_ID \
+       --billing-account $BILLING_ACCOUNT_ID
+   ```
+
+1. Set the project as default:
+
+   ```sh
+   gcloud config set project $PROJECT_ID
+   ```
+
+### Configure locations
+
+1. List available build regions:
+
+   ```sh
+   gcloud artifacts locations list
+   ```
+
+1. Pick a region that is [close to you](https://cloud.google.com/artifact-registry/docs/repositories/repo-locations) and store it in a shell variable:
+
+   ```sh
+   REGION=<NEARBY_REGION>
+   ```
+
+1. List available deployment zones:
+
+   ```sh
+   gcloud compute zones list
+   ```
+
+1. Pick a zone that is [close to you](https://cloud.google.com/compute/docs/regions-zones#available) and store it in a shell variable:
```

Comment:
Not necessarily "close to you" but "close to your users" (same for above)

Reply:
Addressed in f65f664